trts/sys/build.rs

```diff
@@ -2,26 +2,42 @@
 
 //! Builds the FFI function bindings for trts (trusted runtime system) of the
 //! Intel SGX SDK
-use bindgen::Builder;
 use cargo_emit::{rustc_link_lib, rustc_link_search};
 
+const TRTS_FUNCTIONS: &[&str] = &[
+    "sgx_accept_forward",
```

Comment:
Are we sure we want to generate bindings for `sgx_accept_forward`? It's in the `trts.a` but the header which declares it (`trts_inst.h`) is under the `internal/` folder.

Reply:
good catch, removed